src/main/webapp/less/awesome-bootstrap-checkboxcss.less

```diff
@@ -0,0 +1,317 @@
+.checkbox {
```

Comment:
Hey @ChristianMurphy,
In fact originally it was my first willing. But as this comment indicates this warning.
[comments in feedback.less](https://github.com/cousquer/FeedbackPortlet/blob/44a6f3859f60a5c093fe0a5e227f29db6eae9728/src/main/webapp/less/feedback.less#L38)
It's a bad idea to include this small CSS librairy into the resource-server. In the uPortal context, the css properties of this librairy need to [be scoped](https://github.com/cousquer/FeedbackPortlet/blob/44a6f3859f60a5c093fe0a5e227f29db6eae9728/src/main/webapp/less/feedback.less#L45) inside the scope of the portlet like this. 
Because if Feedback-submit.jsp with its colored checkboxes is rendered in a dashboard view (several portlets in the same DOM) without the portlet classes scoping. if another portlet with a checkbox in it, is displayed next or after Feedback-submit, its checkboxes won't be operable...
Trust me I've experienced this with the checkboxes of emailpreview next to Feedback-submit.jsp (->impossible to supress anymore any message into emailpreview...)
Take this for kindly thoughtfulness towards others developers. I do not want the code, which I produce, to disturb their code...